cfgov/cfgov/urls.py

```diff
@@ -404,10 +405,9 @@ def flagged_wagtail_template_view(flag_name, template_name):
 
     flagged_url(
         'REGULATIONS3K',
-        r'^regulations/$',
+        r'^policy-compliance/rulemaking/regulations/',
         lambda request: ServeView.as_view()(request, request.path),
-        fallback=RedirectView.as_view(
-            url='/policy-compliance/rulemaking/', permanent=False),
+        fallback=page_not_found,
```

Comment:
I particularly like the use of `page_not_found`. I think I'd have probably `lambda`ed a `HttpResponseNotFound`, which would have not been the right thing to do.